### ISSUE TITLE
USA Band

### ISSUE THREAD

**Issue by mailyspamy** [alex-kas/nec_terrain#6]
Hello,
I switched a Band on USA Band and now the phone is without signal. Do you have any solution to this problem?
Till now I found that I have to switch to Diag Mode and than use a tool QPST, QXDM and MZTOOL. But I can switch only to Download Mode 9008 and nothing else. Any help will be welcome.
Thank you

**Comment by alex-kas** [comment]
Well, you put your phone in THE BIG A-HOLE, really. I'm not aware of a decent method of going out of this.
For me, I have cured this by flashing copies of modem1 and modem2 partitions taken BEFORE I switched to the USA band. If you have these copies, you are saved!
Otherwise:
You can try, even tough not guaranteed, to null down partitions modem1 and modem2, i.e. fill them in with zeros.
However, most likely, you will end up LOCKED and will have to enter the unlock code to be able to use the device on whatever apart from att.
Some guy on a Russian forum 4pda.ru has reported that you can restore at least pure gsm functionality if you boot your phone with a genuine att sim-card.
As the last resort I can send you some copies of modem1 and modem2 partitions from one of mine unlocked terrains. You can try to flash them on yours. Not sure this will cure yours, as these partitions have intrinsic notion about IMEI and other parameters. The discrepancy (for example, my imei inside modem1,2 data and your imei in the imei-preserving chip) will most likely lead again to the LOCKED state.

Well, if you WILL find a way using whatever, say QPST, please, let me now, it will be indeed an interesting development.
Best, Alex

**Comment by mailyspamy** [comment]
Hello,
so I erased modemst1 and modemst2 and the phone was of course locked, but I had the unlock code so the phone is now full working. The solution with QPST did not worked because I was not able to get the phone to diag mode or I did not find the right drivers for QPST.
But how this whole situation has begin? My wifi is very unstable, it offen shows the 4G icon for a second and the most time is wifi icon grey and not blue. That means that the connection to internet does not work. Do you have the same problem with yours phone? Because so is the phone unusable on wifi connection.
Anyway thank you very much for the idea with modemst and for your willingness to send me your backup data.
Best regards
standal

**Comment by alex-kas** [comment]
Great, that you have the unlock code. Actually, I tried to crack the unlocks but could not. Do you mind to send me your code and your imei. Use my private e-mail for this for security. it is
alex-kas at altaray dot net
About wifi, I had problems in some locations but not always. Generically things work for me.

**Comment by mailyspamy** [comment]
Yes I can send you the IMEI and unlock code. But you have to promise me that you do not give it a third person.  And sometime tell me if you made a progress :)
Would not be easier to compare modemst partitions before unlock and after? It could be, that only one bit or byte is set and the phone will be unlocked. Minimal now we can say that imei is not stored on modemst partitions, just only the information abou lock.
